javascript/ql/src/Security/CWE-640/HostHeaderPoisoningInEmailGeneration.qhelp

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE qhelp PUBLIC "-//Semmle//qhelp//EN" "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>
+Using the HTTP Host header to construct a link in an email can facilitate phishing attacks and leak password reset tokens.
+A malicious user can send an HTTP request to the targeted web site, but with a Host header that refers to his own web site.
+This means the emails will be sent out to potential victims, originating from a server they trust but with
```

Comment:
Suggesting adding a comma between "they trust" and "but with links", purely for better clarity.